CSS/style.css

```diff
@@ -145,9 +186,36 @@ footer {
 .author-text {
   display: block;
   float: left;
+  font-size:
 }
 
-#share-recipe {
+.share-recipe {
   display: inline;
+  width: 30%;
+  height: 55px;
   float: right;
```

Comment:
Remove this float and it should center with the text-align below. With padding it will look better.